thrift/src/main/thrift/client.thrift

```diff
@@ -331,11 +331,24 @@ struct TSLastDataQueryReq {
   3: optional i32 fetchSize
   4: required i64 time
   5: required i64 statementId
-  6: optional bool enableRedirectQuery;
-  7: optional bool jdbcQuery;
+  6: optional bool enableRedirectQuery
+  7: optional bool jdbcQuery
   8: optional i64 timeout
 }
 
+struct TSAggregationQueryReq {
+  1: required i64 sessionId
+  2: required i64 statementId
+  3: required list<string> paths
+  4: required list<string> aggregations
```

Comment:
use thrift-defined enum instead of string

Reply:
fixed